packages/react-dom/src/events/__tests__/ChangeEventPlugin-test.internal.js

```diff
@@ -11,212 +11,219 @@
 
 var React = require('react');
 var ReactDOM = require('react-dom');
-var ReactTestUtils = require('react-dom/test-utils');
-// TODO: can we express this test with only public API?
-var ChangeEventPlugin = require('../ChangeEventPlugin').default;
-var inputValueTracking = require('../../client/inputValueTracking');
 
 function getTrackedValue(elem) {
-  var tracker = inputValueTracking._getTrackerFromNode(elem);
-  return tracker.getValue();
+  return elem.value;
 }
 
-function setTrackedValue(elem, value) {
-  var tracker = inputValueTracking._getTrackerFromNode(elem);
-  tracker.setValue(value);
-}
-
-function setUntrackedValue(elem, value) {
-  var tracker = inputValueTracking._getTrackerFromNode(elem);
-  var current = tracker.getValue();
+var setUntrackedChecked = Object.getOwnPropertyDescriptor(
+  HTMLInputElement.prototype,
+  'checked',
+).set;
 
-  if (elem.type === 'checkbox' || elem.type === 'radio') {
-    elem.checked = value;
-  } else {
-    elem.value = value;
-  }
-  tracker.setValue(current);
-}
+var setUntrackedValue = Object.getOwnPropertyDescriptor(
+  HTMLInputElement.prototype,
+  'value',
+).set;
 
 describe('ChangeEventPlugin', () => {
-  it('should fire change for checkbox input', () => {
+  var container;
+
+  beforeEach(() => {
+    container = document.createElement('div');
+    document.body.appendChild(container);
+  });
+
+  afterEach(() => {
+    document.body.removeChild(container);
+    container = null;
+  });
+
+  fit('should fire change for checkbox input', () => {
```

Comment:
I wrote "fit" in my example to force only a few tests to execute. It should not be used in committed code.

Reply:
Should it just be `it`?

Reply:
Yes.